qiita_db/support_files/patches/33.sql

```diff
@@ -260,7 +265,7 @@ BEGIN
         VALUES ('Demultiplexed', 'Demultiplexed and QC sequeneces')
         RETURNING artifact_type_id INTO demux_type_id;
     INSERT INTO qiita.artifact_type (artifact_type, description)
-        VALUES ('BIOM table', 'Biom table')
+        VALUES ('BIOM', 'Biom table')
```

Comment:
Is it too late to change Biom -> BIOM? entirely not blocking ... but you know :laughing: